mpqp/execution/providers/google.py

```diff
@@ -298,7 +298,7 @@ def extract_result_SAMPLE(
 
     data = [
         Sample(
-            bin_str="".join(map(str, state)),
+            bin_str="".join(map(bin, state)),
```

Comment:
bin adds "0b" at the beginning, is that correct ?

Reply:
yes, in Sample we do :  `int('0b100', 2)` so it works